samples/Bluetooth/BluetoothLESample3/Services/CurrentTimeService.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Threading;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+using nanoFramework.Runtime.Native;
+
+namespace nanoFramework.Device.Bluetooth.Services
+{
+    /// <summary>
+    /// Current Time Service.
+    /// <para>
+    /// Exposes the current time of the device to a connected client.
+    /// A client can subscribe to notification and an updated time will be notified every 1 minute.
+    /// </para>
+    /// <para>
+    /// If writes are allowed the time on the device can be updated.
+    /// </para>
+    /// </para>
+    /// This implementation doesn't support the optional "Local Time Information" or "Reference Time Information" characteristics.
+    /// </para>
+    /// </summary>
+    class CurrentTimeService
```

Comment:
```suggestion
    private class CurrentTimeService
```

Reply:
Can't have that private class with public constructor